tsl/src/hypercore/hypercore_handler.c

```diff
@@ -1735,6 +1748,7 @@ whole_segment_delete_state_create(const HypercoreInfo *hinfo, Relation crel, Com
 	bool isnull;
 	Datum d;
 
+	HypercoreInfo *hinfo = RelationGetHypercoreInfo(rel);
```

Comment:
This is only used in one place below. Maybe move it down there. Could even do: 

```
 heap_getattr(&tp, RelationGetHypercoreInfo(rel)->count_cattno, RelationGetDescr(crel), &isnull);
```

directly.